stubs/requests/requests/compat.pyi

```diff
@@ -16,3 +17,10 @@ from urllib.request import getproxies as getproxies, parse_http_list as parse_ht
 is_py2: bool
 is_py3: bool
 has_simplejson: bool
+
+builtin_str: TypeAlias = str  # noqa: Y042
+# str = str omitted
+# bytes = bytes omitted
```

Comment:
`builtin_str` doesn't need to be annotated with `TypeAlias`; we can use a simple assignment here, and then we don't need the `# noqa`. Also, why are you omitting the `str` and `bytes` re-exports?

```suggestion
builtin_str = str
str = str
bytes = bytes
```

Reply:
Because these were explicitly in the upstream

Reply:
Oops, I misread your comment. 

I was getting errors in my editor that I couldn't do that. So I omitted them because of the error. If it's actually valid though, happy to do that. I think it's because I was trying to use `TypeAlias`

When I hadn't used `TypeAlias` it had been triggering an error locally, but I'd need to pull that up to find it